index.html

```diff
@@ -1808,78 +1877,87 @@ <h2>Authentication</h2>
     </section>
 
     <section>
-      <h2>
-Authorization and Delegation
-      </h2>
-
+      <h3>assertionMethod</h3>
       <p>
-Authorization is the mechanism used to state how operations may be performed on
-behalf of the <a>DID subject</a>. Delegation is the mechanism that the subject
-uses to authorize others to act on their behalf.
+<code>assertionMethod<code> is a <a>verification relationship</a> which an entity can use to indicate that a <a>verification method</a> is authorized to assert a statement on behalf of the <a>DID subject</a>. The <em>verifier</em> of an <code>assertionMethod<code> attempt can check proof of a statement asserted on behalf of the <a> Did subject</a> by checking if the <a>verification method</a> used with the proof is contained in the <code>assertionMethod<code> of the <a>DID Document</a>.
       </p>
 
-      <p class="note" title="Authorization vs authentication">
-Note that Authorization is separate from <a href="#authentication"></a>.
-This is particularly important for key recovery in the case of key loss,
-when the subject no longer has access to their keys, or key compromise,
-where the <a>DID controller</a>'s trusted third parties need to override
-malicious activity by an attacker. See Section
-<a href="#security-considerations"></a>.
-      </p>
+      <p class="note" title="Uses of assertionMethod">
+If assertionMethod is established, it is up to the verifying party to validate that the <a>verification method</a> used for providing proof of an assertion is valid and is contained in the assertionMethod section. An example of how this might be used, is when a verifier validates a verifiable credential has been signed on behalf of the <a>DID Subject</a> to issue a verifiable credential. In this case, the verifier would want to check the <a>verification method</a> used to assert proof on behalf of the <a>Did Subject</a> is contained within the assertionMethod section of the <a>DID Document</a>.
```

Comment:
```suggestion
If <code>assertionMethod</code> is established, it is up to the <a>verifier</a> to validate that the <a>verification method</a> used for providing proof of an assertion is valid and is associated with the <a>assertionMethod</a> <a>verification relationship</a>. An example of when this property is useful is during the processing of a <a>verifiable credential</a> by a <a>verifier</a>. During validation, a <a>verifier</a> checks to see if a <a>verifiable credential</a> has been signed by the <a>DID Subject</a> by checking that the <a>verification method</a> used to assert the proof is associated with the <code>assertionMethod</code> property in the corresponding <a>DID Document</a>.
```

Reply:
Should the verifier term be emphasized or hyperlinked in this case? I'm assuming hyperlink is intentional in this case because it's a defined term in the verifiable credential specification, where as the other uses of verifier are talking only about verification of a verifiable relationship.